wrapper_module_0.pyx

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+cimport numpy as np
+
+cdef extern from 'wrapped_code_0.h':
+    void test(double x, double y, double *z)
+
+def test_c(double x, double y):
+
+    cdef double z = 0
+    test(x, y, &z)
+    return z
```

Comment:
Why are these changes made here? Do they have any relevance?

Reply:
I am so sorry, that file was created automatically and I forgot to delete it

Reply:
I'll make the changes soon